lib/pbench/server/api/resources/endpoint_configure.py

```diff
@@ -0,0 +1,105 @@
+from flask_restful import Resource, abort
+from flask import jsonify
+
+from pbench.server.api.resources.query_apis import get_index_prefix
+
+
+class EndpointConfig(Resource):
+    """
+    EndpointConfig API resource: this supports dynamic dashboard configuration
+    from the Pbench server rather than constructing a disconnected dashboard
+    config file.
+    """
+
+    def __init__(self, config, logger):
+        """
+        __init__ Construct the API resource
+
+        Args:
+            config (PbenchServerConfig): server config values
+            logger (Logger): message logging
+        """
+        self.logger = logger
+        self.host = config.get("pbench-server", "proxy_host")
```

Comment:
The `"proxy_host"` argument is a key, not a default value...right?  What gets returned if the key is missing (or blank)?  Should there be some sanity checking of the config file values?

Reply:
Doing it piecemeal would move us forward in steps.  ;-)

(In any case, now we have #2125.)